test/rmarkdown.md

```diff
@@ -0,0 +1,27 @@
+# R Markdown TEST
```

Comment:
Could you add these to the spec tests?

I've created a proprietary spec format, which describes grammar in a CSS-like way. You can find the R Markdown tests [here](https://github.com/burodepeper/language-markdown/blob/master/spec/fixtures/flavors/rmarkdown.ass). You run the Atom test suite via <kbd>cmd+alt+ctrl+P</kbd> (that's the shortcut for Mac).